README.md

```diff
@@ -30,6 +30,7 @@ The plugin will automatically use any of the following configuration variables,
 * `name` - The title of the site, e.g., "My awesome site"
 * `description` - A longer description of what your site is about, e.g., "Where I blog about Jekyll and other awesome things"
 * `url` - The URL to your site, e.g., `http://example.com`. If none is provided, the plugin will try to use `site.github.url`.
+* `feed_path` - The path to the generated feed, e.g., `atom.xml`. If none is provided, the plugin will default to `feed.xml`.
```

Comment:
What do you think about:

```yml
feed:
  path: `atom.xml`
```

This will create a feed namespace (similar to other plugins), in case we have other options in the future.

I may also encourage the default a bit more. Reading it now, I feel like I have to make a decision about what the "best' feed path is. I'd make it clear that the only time you might use this would be if you had an existing feed, and are using GItHub pages, and don't want to break existing subscribers.

How about a separate section with something like

> # Already have a feed path?
>
> Do you already have an existing feed someplace other than `/feed.xml`, but are on a host like GitHub Pages that doesn't support machine-friendly redirects? If you simply swap out `jekyll-feed` for your existing template, your existing subscribers won't continue to get updates. Instead, you can specify a non-default path via your site's config.

```yml
feed:
  path: atom.xml
```

To note, you shouldn't have to do this unless you already have a feed you're using, and you can't redirect existing subscribers.